libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/RouteProgress.kt

```diff
@@ -41,10 +41,9 @@ class RouteProgress private constructor(
     fun distanceTraveled(): Float = distanceTraveled
 
     /**
-     * Provides the duration remaining in seconds till the user reaches the end of the route.
+     * Provides the duration remaining in milliseconds till the user reaches the end of the route.
```

Comment:
actually didn't realize that it was the documentation that threw me off too
https://github.com/mapbox/mapbox-navigation-android/pull/2569/files#diff-d63e04e6e9257f3b5b59fcaef29b39b6R49